src/main/java/ee/tuleva/onboarding/conversion/UserConversionService.java

```diff
@@ -182,11 +183,9 @@ private boolean isTransfersComplete(
   }
 
   private boolean isConvertedFundManager(Exchange exchange) {
-    return exchange
-        .getTargetFund()
-        .getFundManager()
-        .getName()
-        .equalsIgnoreCase(CONVERTED_FUND_MANAGER_NAME);
+    FundDto targetFund = exchange.getTargetFund();
+    return targetFund != null
+        && targetFund.getFundManager().getName().equalsIgnoreCase(CONVERTED_FUND_MANAGER_NAME);
```

Comment:
According to the [law of demeter](https://en.wikipedia.org/wiki/Law_of_Demeter#In_object-oriented_programming) we should ask for the fund manager name directly from the targetFund. I.e. refactor it in a way that we would do: `targetFund.getFundManagerName()` instead.